### PR TITLE
add(tools): tailwindcsscolors.com, tailwindgradients.com and fullwindcss.com

### PR DESCRIPTION
**Pull request by ErwinAI**
| Name                 | Link                 |
| -------------------- | -------------------- |
| TailwindCssColors | tailwindcsscolors.com |
| TailwindGradients | tailwindgradients.com |
| Fullwind CSS| fullwindcss.com |

Color palete cheatsheet, gradient collection and extended shades for the default palette.

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] My item is in line with the [Tailwind brand usage guidelines](https://tailwindcss.com/brand#usage)
- [x] I have read and followed the [contribution guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,6 +120,9 @@
 - 🎨🌍🔧 [Palettolithic](https://palettolithic.com) - Generates harmonius color palettes based on one color.
 - 🎨🌍🔧 [Tailwind Gradient Generator](https://tailwindcomponents.com/gradient-generator) - Create perfect Tailwind CSS gradients with zero lines of code.
 - 🎨🌍🔧 [Tints](https://www.tints.dev/) - Color palette penerator and API for Tailwind CSS.
+- 🎨🌍🔧 [Tailwind CSS Colors](http://tailwindcsscolors.com) - Cheat sheet for Tailwind Colors with easy copying in various formats.
```
Comment (innocenzi) [inline README.md:123]:
```suggestion
- 🎨🌍🔧 [Tailwind CSS Colors](http://tailwindcsscolors.com) - Cheat sheet for Tailwind CSS colors.
```

File: README.md [hunk 2]
```diff
@@ -120,6 +120,9 @@
...
+- 🎨🌍🔧 [Tailwind Gradients](http://tailwindgradients.com) - Collection of Tailwind gradients, easy to browse and one-click copy.
```
Comment (innocenzi) [inline README.md:124]:
```suggestion
- 🎨🌍🔧 [Tailwind Gradients](http://tailwindgradients.com) - Collection of gradients using Tailwind CSS classes.
```

File: README.md [hunk 3]
```diff
@@ -120,6 +120,9 @@
...
+- 🎨🌍🔧 [Fullwind CSS](https://fullwindcss.com/) - Additional shades for the Tailwind CSS Color palette.
```
Comment (innocenzi) [inline README.md:125]:
```suggestion
- 🎨🌍🔧 [Fullwind CSS](https://fullwindcss.com/) - Extend Tailwind CSS color palettes with additional shades.
```

**Review by innocenzi — CHANGES_REQUESTED** [review]

**Review by innocenzi — APPROVED** [review]

**Comment by laurentlahmy** [comment]
😍 Thanks Erwin!

**Pull request merged**